dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/ownership/TrackerOwnershipController.java

```diff
@@ -97,15 +98,16 @@ public WebMessage updateTrackerProgramOwner(
             "trackedEntityInstance", trackedEntityInstance, "trackedEntity", trackedEntity);
 
     trackerOwnershipAccessManager.transferOwnership(
-        trackedEntityService.getTrackedEntity(trackedEntityUid),
+        trackedEntityService.getTrackedEntity(
+            trackedEntityUid, UID.of(program), TrackedEntityParams.TRUE),
         programService.getProgram(program),
         organisationUnitService.getOrganisationUnit(ou));
     return ok("Ownership transferred");
   }
 
   @PostMapping(value = "/override", produces = APPLICATION_JSON_VALUE)
   @ResponseBody
-  public WebMessage overrideOwnershipAccess(
+  public WebMessage grantTemporaryAccess(
```

Comment:
Does this mean the path `/override` is also not accurate?

Reply:
From a strict RESTful naming convention, I guess having a verb is odd for the API path. APIs could have been `api/tracker/ownership/permanent` and `api/tracker/ownership/temporary`

But apart from that, `override` is not inaccurate.  The functionality for this API is called "Breaking the glass" which is pretty much overriding/bypassing existing access check mechanisms. Atleast for now, I guess we really don't have to change API paths for this. Unless there is very strong reasonings, which I don't see at this point.

Reply:
I'm fine with not renaming the path for now, but I'd stick to the new method name.
IMO, `override` means replacing the existing value with a new one, and that's not the case here, is it?
Here we simply grant temporary access to someone who's not in the CS, but everyone who is in the CS still can see this enrollment, so their access is not overridden.

Reply:
Replacing existing value with a new one is `overwrite`. 

`Override` in Security/access control only means, for example, to allow a user to access something which in normal scenario they are not allowed to access. This is exactly what we do when saying granting temporary access.